### PR TITLE
combine loading into one component (ActivityIndicator)

### PR DESCRIPTION
**Pull request by teyou**

### CODE REVIEW

File: loading.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ export default class Loading extends React.Component {
 
   render () {
     return (
-      <ActivityIndicatorIOS
+      <ActivityIndicator
         color={this.props.color}
         ref={c => this._root = c}
-        size="small"
+        size="large"
```
Comment (RealOrangeOne) [inline loading.js]: was this intentional? If so, why?

**Comment by RealOrangeOne** [comment]
Nice! I'd been meaning to do this change myself. Although, now the `Loading` component is simply a dumb wrapper around `ActivityIndicator`, I think it might be nicer to remove `loading.js` and just have it inside the main component?

File: loading.js (hunk 1 above)
Comment (teyou) [inline loading.js]: ops. nope.  forgot to undo

**Comment by teyou** [comment]
That would be even better! 
Maybe you can delete my PR and put in the loading logic inside index.js

**Comment by RealOrangeOne** [comment]
Could you do it as part of your PR? seeing as it's all mostly related?

**Comment by RealOrangeOne** [comment]
Perfect update! Once you fix the tests (it's the linter), then i'll merge it!

**Comment by teyou** [comment]
ops. my bad. my machine got issue with the eslint npm package

**Pull request merged**